ui/app/templates/vault/cluster.hbs

```diff
@@ -3,6 +3,37 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 <Sidebar::Nav::Cluster />
+{{#if this.auth.currentToken}}
+  {{#if this.customMessages.bannerMessages.length}}
+    {{#each this.customMessages.bannerMessages as |bannerMessage|}}
```

Comment:
I think the wrapping `{{if}}` statement is unecessary, because if the bannerMessages are empty it won't show any of them. But worth checking with various falsey values

Reply:
Good point! The only values that need a falsey values check is the link attribute since title and description and required if custom messages exists. Will tackle that in my next ticket :)